frontend/src/libraries/wamp/types.ts

```diff
@@ -0,0 +1,579 @@
+export type TransactionCountHistory = {
+  date: string;
+  total: number;
+};
+
+export interface NodeInfo {
+  ipAddress: string;
+  accountId?: string;
+  nodeId: string;
+  lastSeen: number;
+  lastHeight: number;
+  agentName: string;
+  agentVersion: string;
+  agentBuild: string;
+  status: string;
+  latitude: number;
+  longitude: number;
+  city?: string;
+}
+export type StakingStatus =
+  | "active"
+  | "joining"
+  | "leaving"
+  | "proposal"
+  | "idle"
+  | "newcomer"
+  | "on-hold";
+
+export interface BaseValidationNodeInfo {
+  account_id: string;
+  is_slashed?: boolean;
+  num_produced_blocks?: number;
+  num_expected_blocks?: number;
+  public_key?: string;
+  currentStake: string;
+  proposedStake?: string;
+  cumulativeStakeAmount?: string;
+  stakingStatus?: StakingStatus;
+  networkHolder?: boolean;
+  shards?: [number];
+  nodeInfo?: NodeInfo;
+}
```

Comment:
At some point we will need to make a decision about the naming and enforce it. I prefer to use snake case when passing data over the wire and then client code should just map it over to the style convention on the receiving side, but it is cumbersome and we can argue that Explorer backend is just an extension of Explorer frontend and RPC being a procedure call can follow a single convention of the code-base.

@luixo I feel that at this point whichever strategy is better than this mix of snake-case and camel-case fields floating around.

Reply:
In this PR I tried to make as little excessive modifications as I could, and changing casing is one of those.
At some point I'll introduce linting with casing rules so we'll never have to argue on that anymore :)